lib/features/accounts/presentation/pages/create_account/create_account_type.dart

```diff
@@ -51,13 +53,25 @@ class _Item extends StatelessWidget {
       case CreateType.mnemonic:
         context.router.push(ImportMnemonicFormRoute());
         break;
+      case CreateType.rawseed:
+        //TODO How does this generated navigator work?
```

Comment:
Please follow this to generate path using auto_route:
1. Run `packages pub run build_runner build --delete-conflicting-outputs`
2. Add generated path to `lib/features/accounts/router/create_account_page_route.dart`
3. Use `context.router.push(**);` to navigate